app/main.py

```diff
@@ -1 +1,43 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
You are using an extra `else` statement in the `feed` method of the `Animal` class. According to the checklist, it's better to avoid unnecessary `else` statements. You can simply return 0 at the end of the method. It will be returned if `self.is_hungry` is `False`.